dev/com.ibm.ws.security.jaspic_fat/fat/src/com/ibm/ws/security/jaspic11/fat/audit/JASPIBasicAuthenticationAuditTest.java

```diff
@@ -56,6 +64,10 @@
 @Mode(TestMode.FULL)
 public class JASPIBasicAuthenticationAuditTest extends JASPITestBase {
 
+    @ClassRule
+    public static RepeatTests r = EERepeatActions.repeat("com.ibm.ws.security.jaspic11.fat.audit",
+    EERepeatActions.EE11).with(new EmptyAction().fullFATOnly()).andWith(new FeatureReplacementAction(AuditCommonTest.AUDIT_1_0_FEATURES, AuditCommonTest.AUDIT_2_0_FEATURES));
+
```

Comment:
```suggestion
    /**
     * Need the first repeat to make sure that audit-2.0 from a previous repeat gets put back to audit-1.0
     */
    @ClassRule
    public static RepeatTests auditRepeat = RepeatTests.with(new FeatureReplacementAction("audit-2.0", "audit-1.0"))
                    .andWith(new FeatureReplacementAction("audit-1.0", "audit-2.0"));

```
Don't need the EE repeat since it is already done in FATSuite.